main.go

```diff
@@ -182,18 +182,17 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
-	re := regexp.MustCompile(`artifacttype:\\s.*`)
+	re := regexp.MustCompile(`artifactType:\s.*`)
 	artifactTypematch := re.FindString(plan)
-	artifactType := yamlsTargetArtifactTypeValue
-	if strings.HasSuffix(artifactTypematch, knativeTargetArtifactTypeValue) {
-		artifactType = knativeTargetArtifactTypeValue
-	} else if strings.HasSuffix(artifactTypematch, helmTargetArtifactTypeValue) {
-		artifactType = helmTargetArtifactTypeValue
+	var artifactType string
+	if artifactTypematch == "" {
+		artifactType = ""
+	} else {
+		match := []rune(artifactTypematch)
+		name := string(match[len("artifactType:"):len(match)])
+		artifactType = strings.Replace(name, " ", "", -1)
 	}
+
```

Comment:
How about something like 

```
var artifactType
re := regexp.MustCompile(`artifactType:\s*([^\s]*)`)
matches = rgx.FindStringSubmatch(plan)
if len(matches) > 0 {
    artifactType = matches[0]
}
```

Reply:
Hmm,that's a smart solution. But is [^\s]* necessary, since \s* should be able to catch all white spaces behind : I think?

Reply:
It will help ignore the space at the end. Also now that I think about it a + is better than a * there

Reply:
I made some changes, please take a look. [^\s]* would cause no match if there is any whitespace in the latter part. So I used (.+) instead, and then removed any possible whitespace in it.